website/content/docs/configuration/worker/pki-worker.mdx

```diff
@@ -73,6 +73,22 @@ kms "aead" {
 }
 ```
 
+## Session Recording <sup>Enterprise and HCP Only</sup>
+Session Recording requires at least one PKI worker with access to local and remote storage.
+PKI Workers used for session recording require an accessible directory defined by `recording_storage_path` for
+storing in-progress session recordings. On session closure, a local session recording is moved to remote storage and
+deleted locally.
+
+Example (not safe for production!):
```

Comment:
Do we use this sort of language elsewhere? It feels a little unprofessional. How about
```suggestion
Development example:
```